docs/application/native/guides/app-management/app-controls.md

```diff
@@ -808,6 +812,50 @@ The following common application controls are available:
 - [Voice Recorder](common-appcontrols.md#voice-recorder)
 - [VPN Service](common-appcontrols.md#vpn-service)
 
+<a name="URI"></a>
+## Application Control URI
+
+> **Note**
+>
+> Application Control URI is supported since Tizen 5.5.
+
+[Application Control URIs](../../api/mobile/5.5/group__CAPI__APP__CONTROL__URI__MODULE.html) provides functions to set, get, build URI data conform to [RFC 3986](https://tools.ietf.org/html/rfc3986).
+
+- Syntax
+
+   URI = scheme:[//authority]path[?query][#fragment] <br/>
+   authority = [userinfo@]host[:port]
```

Comment:
Should this be in code style? 
What is your opinion on the same?
   ```
    URI = scheme:[//authority]path[?query][#fragment] <br/>
    authority = [userinfo@]host[:port]
   ````
If you dont think, it should be in code style, ignore the comment.

Reply:
It doesn't matter.